include/tudat/astro/observation_models/observationModel.h

```diff
@@ -418,6 +421,39 @@ class ObservationModel
         }
     }
 
+    TimeType computeBiasedObservationTime( const double observationTime )
```

Comment:
We can keep this for now, but doing dynamic casts in the inner loop of the simulation should be avoided, as they are quite costly. Once the observation model is created, you can't add any new biases to it, right? That should means that any time conversion function can already be defined at the moment the observation model is created, and the dynamic casts can be done then.